src/bumper/bumper_test.go

```diff
@@ -16,10 +17,19 @@ type testCase struct {
 	expected  *semver.Version
 }
 
-func TestBumper_Bump(t *testing.T) {
+type testCaseWithCap struct {
+	name          string
+	changelog     changelog.Changelog
+	current       *semver.Version
+	expected      *semver.Version
+	entryCap      bump.Type
+	dependencyCap bump.Type
+}
+
+func TestBumper_Bump_changesOnlyCases(t *testing.T) {
```

Comment:
Is there a reason to have so many, yet so similar test functions? It seems a bit confusing, or tedious to keep those functions similar.

Reply:
Go linter did not like the length of the functions: https://github.com/newrelic/release-toolkit/pull/128/commits/9fe51a8bd8f95282a614f1f40eabb31e7eca4246#diff-b724875f4a504c8deb98d07b6ff7a128127f4c3d49220ae899e68863f7cf7799R29

So I split them in three. But I can revert that part and add a `nolint` clause.

Reply:
I see there are more `nolint` on other tests so I am adding them to these tests too...